assets/src/components/cloud-setup/steps/scm-setup/GitHubSetupStep.tsx

```diff
@@ -62,43 +86,101 @@ export function GitHubSetupStep() {
       gap="medium"
     >
       {registerGhAppError && <GqlError error={registerGhAppError} />}
-      <Body1P>
-        Plural Console is best with a Git connection. You can either use the
-        pre-made Plural GitHub application or install your own.
-      </Body1P>
-      <TabList
-        stateRef={tabStateRef}
-        stateProps={{
-          orientation: 'horizontal',
-          selectedKey: setupType,
-          onSelectionChange: (key) => setSetupType(key as SetupType),
-        }}
-      >
-        <StretchedTabSC key={SetupType.GhApp}>Install on GitHub</StretchedTabSC>
-        <StretchedTabSC key={SetupType.Manual}>
-          Manual installation
-        </StretchedTabSC>
-      </TabList>
-      <TabPanel
-        stateRef={tabStateRef}
-        tabKey={setupType}
-      >
-        {setupType === SetupType.GhApp ? (
-          <GitHubAppSetup
-            submitInstallationId={(id) => {
-              // this will trigger the mutation in the effect
-              setParams({ [INSTALLATION_ID_PARAM]: id })
+      {showWebhookSetup ? (
+        <WebhookFormSC
+          onSubmit={(e) => {
+            e.preventDefault()
+            createWebhook()
+          }}
+        >
+          {webhookError && <GqlError error={webhookError} />}
+          <Body1P>
+            Please provide the repository owner so Plural can create a webhook
```

Comment:
This should also be `Please provider the github organization or repository slug so...`